### PR TITLE
Adding skill tester scripts

### PR DESCRIPTION
**Pull request by ig-perez**
I have remove old unit test and add Skill Tester unit test scripts. Readme.md and circle.yml have been updated as well.

### CODE REVIEW

**Comment by jkelvie** [comment]
Is there an end-to-end test in here? Where is it?

File: lib/products.js [hunk 1]
```diff
@@ -66,9 +66,19 @@ const getProductList = function() {
 };
 
 const getProduct = function () {
-    const productList = getProductList();
-    const randIndex = Math.floor(Math.random() * productList.length);
-    return productList[randIndex];
+
+    //°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°° ADDED BY IVAN °°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°
```
Comment (jperata) [inline lib/products.js]: remove the // added by ivan, but let the // for testing purposes

File: lib/products.js [hunk 2]
```diff
@@ -66,9 +66,19 @@ const getProductList = function() {
...
+    //For testing purposes, set the product to guess to Fitbit if the UNIT_TEST environment variable is set
```
Comment (jperata) [inline lib/products.js]: space between // and FOr

File: lib/products.js [hunk 3]
```diff
@@ -66,9 +66,19 @@ const getProductList = function() {
...
+    if (process.env.UNIT_TEST) {
+        return getProductList()[1]
+    }
+    else {
+        const productList = getProductList();
+        const randIndex = Math.floor(Math.random() * productList.length);
+        return productList[randIndex];
+    }
+    //°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°° ADDED BY IVAN °°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°°
```
Comment (jperata) [inline lib/products.js]: remove added by ivan

**Comment by ig-perez** [comment]
After convresation, e2e scripts will be added later.

**Pull request merged**